tests/envalid.ts

```diff
@@ -1,25 +1,29 @@
-import { cleanEnv, makeValidator, str, bool, num, json, url, email, Spec, Specs } from '..';
+import { cleanEnv, makeValidator, str, bool, num, json, url, email, Spec, Specs } from '..'
```

Comment:
Oops, sorry this entire file got churned -- I guess the hook did it.

Reply:
that's fine 🙂